src/main/java/org/pesmypetcare/webservice/controller/appmanager/MyPetCareRestController.java

```diff
@@ -33,7 +34,7 @@
      */
     @PostMapping("/signup")
     @ResponseStatus(HttpStatus.CREATED)
-    public void signUp(@RequestBody Map<String, Object> user) throws DatabaseAccessException, FirebaseAuthException {
+    public void signUp(@RequestBody Map<String, Object> user) throws DatabaseAccessException, FirebaseAuthException, DocumentException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 136).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5641243)